image-swipe.android.ts

```diff
@@ -119,11 +124,6 @@ class ImageSwipePageChangeListener extends java.lang.Object implements android.s
         if (preloadedImageView) {
             preloadedImageView.reset();
         }
-
-        preloadedImageView = owner.android.findViewWithTag("Item" + (index + 1).toString()) as ZoomImageView;
-        if (preloadedImageView) {
-            preloadedImageView.reset();
-        }
```

Comment:
This is not a duplicate. The previous one preloads `index -1` and this one preloads `index+1`